lgc/test/UnlinkedVsGsInputs.lgc

```diff
@@ -1,8 +1,8 @@
 ; Check that after merging the VS and GS shader the result has the vertex input as the last parameter, and it is being passed
 ; to the vertex shader, which expects it as the last parameter.
 ; RUN: lgc -mcpu=gfx900 %s -o /dev/null -print-after=lgc-patch-prepare-pipeline-abi 2>&1 | FileCheck --check-prefixes=CHECK %s
-; CHECK: IR Dump After Patch LLVM for preparing pipeline ABI (lgc-patch-prepare-pipeline-abi)
-; CHECK: IR Dump After Patch LLVM for preparing pipeline ABI (lgc-patch-prepare-pipeline-abi)
+; CHECK: IR Dump After Patch LLVM for preparing pipeline ABI (lgc-patch-prepare-pipeline-abi) 
+; CHECK: IR Dump After Patch LLVM for preparing pipeline ABI (lgc-patch-prepare-pipeline-abi) 
```

Comment:
Trailing spaces?

Reply:
This PR removes formatting changes from the merge (see first comment).
I guess trailing spaces aren't desirable - so we could remove them, or would this be better done as a separate cleanup PR (although that seems unnecessarily complicated)?